### ISSUE TITLE
Content not showing up in the _index.md and index.md

### ISSUE THREAD

**Issue by BVauer** [2-REC/hugo-myportfolio-theme#10]
HI,
Thank you for this template which is very nice ! It works perfectly for me for the home page. The problem at the moment for me is that the things in the index.md and _index.md are not showing up (I mean the content is not showing up, only the elements in the front matter appear).
I have this problem with the exampleSite.
I am using mac os 10.11.6 and hugo v0.62.0
I have looked around but can't find the fix. Any help would be greatly appreciated !
Many thanks :)

**Comment by BVauer** [comment]
Ok, so going a bit further I remarked that in the list.md and single.md files, the {{with .Content}} sections don't work for me.
If I replace the section by {{  .Content}} it all works smoothly.
Though I don't get why it does this. Do you have any insight?

**Comment by 2-REC** [comment]
Hello @BVauer 

First, I apologize for the late reply, I haven't been on this repository for some time, and weirdly I didn't get any notification when you created the issue.
Then, I would like to thank you for the interest in my template theme, I'm glad you enjoy it, and hope you could successfully use it.

Regarding the problem you had, maybe it is too late now, but I will still try to answer.

If I am not wrong, I think the issue is that when writing something like:
```
{{ with .Content }}
```
it specifies that the following nested block will be using "Content" as the local context/element (referred to by ".").
So in the following block, "Content" is referred to using the operator "." instead of the full name ".Content".

In our case:
```
{{ with .Content }}
    {{ . }}
{{ end }}
```
is equivalent to:
```
{{ .Content }}
```

So if you only have the "with" block without a nested block and an associated "end" block, nothing will happen (and it is actually syntactically incorrect).


I hope this answers to your question, and that the issue can be closed.
Kind regards,
Derek.

**Comment by PeterPandaluki** [comment]
Having applied the fix above, I am now seeing the following (or similar) page path printed on every page:
Page(/resources/lang/_index.md)
I am unable to find the reason.

**Comment by 2-REC** [comment]
Hello @PeterPandaluki ,

You shouldn't have to apply any fix, as the point explained above happens when there is an error in the markdown parsing (from what I understood).

So by "applying the fix", do you mean you removed the "with" block and its embedded "markdownify" command? (Only have the ".Content" command left?)

If so, I think the problem might be that by removing the "markdownify" command, the content markdown is not parsed anymore, and some data don't get converted as expected.
Is the "Page(...)" command part of the page content?

Without seeing the code it is not easy to determine the problem, so I might be completely wrong... 😅

If you can provide me more information (eg an example of _index.md page), I might be able to help you more.

Best regards,
Derek.

**Comment by PeterPandaluki** [comment]
Hi, I am attaching a _index.md file and the alterations I've made in two layout files.

_index.md
# Template for projects.
title: "Reading"
subtitle: ""
image: "reading.jpg"
small_image: "reading_med.jpg"
#small_url: "<URL>"
categories: [ "resourses" ]
tags: [ "reading", "books" ]
#images_directory; "images"
#images_static: "false"
#images_copyright: "<COPYRIGHT>"
##For "image" image files:
# (Only required for caption and specific copyrights)
#images:
#- src: "<IMAGE_FILE>"
#  title: "<LEGEND>"
#  copyright: "<COPYRIGHT>"
##For "logo" image files:
#resources:
#- src: "logos/<IMAGE_FILE>"
#  title: "<TOOLTIP>"
---

list.html

{{ define "content" }}
    {{ .Content}}
        {{ . | markdownify }}
        <hr>
    {{ end }}

single.html


    {{ .Content}}

**Comment by PeterPandaluki** [comment]
That didn't go so well!

**Comment by PeterPandaluki** [comment]
This is from a page source:

               
                <div class="list-page">
                    <div class="list-page-container">
                        <article class="list-page-content">
                            
                                <hgroup class="title-block">
    
    <h1>Gàidhlig</h1>

    
</hgroup>
                       
    
        Page(/languages/gaidhlig/_index.md)
        <hr>
                        
                        </article>
                    </div>
                </div>
            </main>
        

        
            <footer>

**Comment by 2-REC** [comment]
Hi @PeterPandaluki , (and @BVauer )
Sorry for the late reply ...

I have found the reason of the problem:
Since Hugo version 0.60, there has been a change (undocumented) that by default doesn't allow raw HTML inside a Markdown content.
FYI, the problem is detailled here:
https://discourse.gohugo.io/t/raw-html-getting-omitted-in-0-60-0/22032/6

The solution to avoid this is to add to the config the following lines:
```
[markup.goldmark.renderer]
unsafe= true
```
(or by removing the "markdownify" keyword if don't need to parse it)

I have updated the "config.toml" to fix the issue.


Another thing is in the code you have posted for "list.html", there is an error:
```
{{ define "content" }}
{{ .Content}}
{{ . | markdownify }}
<br>
{{ end }}
```

There should not be the 2 lines "{{ .Content }}" and {{ . | markdownify }}", it should instead be either:
```
{{ with .Content}}
{{ . | markdownify }}
```
or
```
{{ .Content | markdownify }}
```


I am sorry for the inconvenience, and hope all problems will now be fixed.

Thank you,
Derek.
